.github/workflows/post-submit.yaml

```diff
@@ -8,31 +8,31 @@ on:
 
 jobs:
   push_to_registry:
-    name: Push a container image to quay.io/medik8s
+    name: Build and push images to quay.io/medik8s
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
I see in couple of places upgrade of the version.
Can I take it the the github action itself are versioned ?

Reply:
> Can I take it the the github action itself are versioned ?

Yes, see https://github.com/actions/checkout/releases